scipy/ndimage/filters.py

```diff
@@ -1186,3 +1186,102 @@ def generic_filter(input, function, size=None, footprint=None,
     _nd_image.generic_filter(input, function, footprint, output, mode,
                          cval, origins, extra_arguments, extra_keywords)
     return return_value
+
+
+@docfiller
+def line_integral_convolution(input, velocity, kernel, origin=0, order=3, reweight=True):
+    """
+    Line integral convolution of an input image with an arbitrary kernel.
+    Lines are defined by a velocity field.
+
+    Parameters
+    ----------
+    input : array_like
+        Random image to be convolved
+        (can be created for instance with
+        "1. * (np.random.random(shape) > 0.5)")
```

Comment:
you should surround all code examples with double backticks, then they'll render correctly in the html/pdf docs.